daml-script/runner/BUILD.bazel

```diff
@@ -33,6 +33,7 @@ da_scala_library(
         "@maven//:org_typelevel_paiges_core",
     ],
     scalacopts = script_scalacopts,
+    tags = ["maven_coordinates=com.daml:daml-script-runner:__VERSION__"],
```

Comment:
maybe worth dropping runner from the name? I’m not sure it really has much meaning.